lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -35,10 +35,10 @@ static std::optional<Path> getActualModuleMapPath(
 
   Path result;
 
-  StringRef SDKPath = Opts.getSDKPath();
-  if (!SDKPath.empty()) {
-    result.append(SDKPath.begin(), SDKPath.end());
-    llvm::sys::path::append(result, "usr", "lib", "swift");
+  if (!Opts.RuntimeResourcePath.empty()) {
+    result.clear();
```

Comment:
This line needs to be in the second if-condition below. Currently the SDK check would fail because the path would be incorrectly constructed.

Reply:
Oh, I simply copy-pasted this from below, maybe that's why it was failing for them on Windows? I will update this and maybe they can try again.

Reply:
> maybe that's why it was failing for them on Windows

Ah, I'm pretty sure that's indeed the reason! 😃

Reply:
Alright, rebased and made this change.

@egorzhdan, another CI run would be good.

@compnerd or @hyp, please try this updated pull with this change pointed out by Egor and let us know if it works now.

Reply:
@hjyamauchi, maybe you can trigger a Windows build with this pull, if your TBC colleagues are too busy.

Reply:
@finagolfin if you mean the TBC CIs, they are still not in a good state and we're working on fixing them, which may need a bit more time :)

Reply:
@hjyamauchi, I'm talking about whatever was used to produce the TBC error output above, which we don't see on this CI, could be CI or a local run. If your CI is broken now, maybe you can try a local run?

Reply:
I'm not actually sure what he ran there but I'll see if I can do some build

Reply:
I ran a local build at tip of tree with this PR and it was successful. But I didn't build (or know how to build) it with the Android SDK and this was probably not helpful. Unless @hyp knows how, this may need to wait for @compnerd to come back.

Reply:
Thanks @hjyamauchi, that was still a useful signal.